kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -10153,6 +10170,36 @@ public boolean isDropboxMaintainer(String siteId)
 		// (dropbox maintain in their myWorkspace just gives them access to their own dropbox)
 		return m_securityService.unlock(AUTH_DROPBOX_MAINTAIN, m_siteService.siteReference(siteId));
 	}
+	
+	/**
+	 * Determine whether the user has the dropbox.groups permission 
+	 * 
+	 * @return True if user has dropbox.groups permission, false otherwise.
+	 */
+	public boolean isDropboxGroups()
```

Comment:
In isDropboxGroups() method I have copied the logic of isDropboxMaintainer() existing method, just changing the permission that is checked. isDropboxGroups() is not called, but I though that if isDropboxMaintainer had a method without parameters which rely on the placement I should add one too. I'll delete it.